### PR TITLE
File Added : Assignment 7 & 8

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

File: urmila/HTML/Assignment7/Homepage.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment7/Homepage.html]: link tag and title tag is missing add them first

File: urmila/HTML/Assignment7/profile.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <META http-equiv="Content-Type" content="text/html; charset=utf-8">
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment7/profile.html]: write meta in small letters , title , link , meta tags are missing add them

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

File: urmila/HTML/Assignment7/home.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+					<div class="c">
+						<label class="lc"><b>Comments:</b></label>
+						<textarea name="" cols="30" rows="10"></textarea>
+					</div>
+					<button class="btnsubmit">Submit</button>
+				</form>
+			</div>
+		</section>
+	</body>
+</html>
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment7/home.html]: check indentation and make code clear

File: urmila/HTML/Assignment7/home.html [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+				<b class="a">Age:</b>
+				<input type="radio" id="less" name="age" value="number" />
+				<label for="less"><b>less than 18 </b></label>
+				<input
+					type="radio"
+					id="equal"
+					name="age"
+					value="number"
+					checked
+				/>
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment7/home.html]: input's all attributes should be in one line

File: urmila/HTML/Assignment7/home.css [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        margin-bottom: 60px;
+        margin-left: 700px;
+        background-color: black;
+        width:10%;
+    }
+    #sec1, #sec2, #sec3{
+        height: 200px;
+        padding-top: 60px;
+    }
+</style>
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment7/home.css]: add extra line and there is not need of adding style tag in css file remove it

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

File: urmila/HTML/Assignment7/Homepage.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        body {
+            background-image: url("Image.jpg");
+            background-repeat: no-repeat;
+            background-size: cover;
+            background-size: 1000px 1000px;
+            background-attachment: fixed;
+        }
+    </style>
+</head>
+<div>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/Homepage.html]: body tag missing 
make proper indentation 
title is missing for this page

File: urmila/HTML/Assignment7/Login.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    </head>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/Login.html]: title is missing

File: urmila/HTML/Assignment7/Login.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    <body>
+        <style>
+            body {
+                background-color: aqua;
+                background-repeat: no-repeat;
+                background-size: cover;
+                background-size: 1000px 1000px;
+                background-attachment: fixed;
+            }
+        </style>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/Login.html]: this need to be in head tag

File: urmila/HTML/Assignment7/Login.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+                    <div>
+                        <a href="http://login.html" target="_blank"><button>Profile</button></a>
+                        <a href="http://login.html" target="_blank"><button>login</button></a>
+                        <a href="http://signup.html" target="_blank"><button>sign in</button></a>
+                    </div>
+                </div>
+            </div>
+            <div>
+                <p>Please enter your Email and password</p>
+                <form action="http://profile.html" target="_blank" onsubmit='try {return window.confirm("This form may not function properly due to certain security constraints.\nContinue?");} catch (e) {return false;}'>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/Login.html]: you know is this working or not

File: urmila/HTML/Assignment7/home.css [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+        margin-top: 60px;
+        margin-bottom: 60px;
+        margin-left: 700px;
+        background-color: black;
+        width:10%;
+    }
+    #sec1, #sec2, #sec3{
+        height: 200px;
+        padding-top: 60px;
+    }
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/home.css]: add 1 space before { this every where in this file

File: urmila/HTML/Assignment7/home.html [hunk 11]
```diff
@@ -0,0 +1,132 @@
+!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/home.html]: give proper title for this page

File: urmila/HTML/Assignment7/home.html [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+                <label for="mob" class="mob"><b>Contact No.: </b></label>
+                <input type="number" id="mob" placeholder="Contact Number" /><br /><br />
+                <b class="g">Gender:</b>
+                <input type="radio" id="male" name="gender" value="male" />
+                <label for="male"><b>Male </b></label>
+                <input type="radio" id="female" name="gender" value="female" />
+                <label for="female"><b>Female </b></label>
+                <input type="radio" id="Other" name="gender" value="other" />
+                <label for="male"><b>Other</b></label
+                ><br /><br />
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/home.html]: make sure closing of tag

File: urmila/HTML/Assignment7/home.html [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+                    <div class="c">
+                        <label class="lc"><b>Comments:</b></label>
+                        <textarea name="" cols="30" rows="10"></textarea>
+                    </div>
+                    <button class="btnsubmit">Submit</button>
+                </form>
+            </div>
+        </section>
+    </body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/home.html:133]: give proper class and id

File: urmila/HTML/Assignment7/profile.html (hunk 2 above)
Comment (Pawan1005) [inline urmila/HTML/Assignment7/profile.html]: add title here

File: urmila/HTML/Assignment7/profile.html [hunk 14]
```diff
@@ -0,0 +1,122 @@
...
+<body>
+    <style>
+        body {
+            background-color: aqua;
+            background-repeat: no-repeat;
+            background-size: cover;
+            background-size: 1000px 1000px;
+            background-attachment: fixed;
+        }
+    </style>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/profile.html]: use this in head tag

File: urmila/HTML/Assignment7/profile.html [hunk 15]
```diff
@@ -0,0 +1,122 @@
...
+                            <div>
+                                <button type="submit"> submit</button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/profile.html]: make proper indentation

File: urmila/HTML/Assignment7/sign.html [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+    <style>
+        body {
+            background-color: aqua;
+            background-repeat: no-repeat;
+            background-size: cover;
+            background-size: 1000px 1000px;
+            background-attachment: fixed;
+        }
+    </style>
+    <div>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/sign.html]: closing of head tag and opening of head tag is missing

File: urmila/HTML/Assignment7/sign.html [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+                <p>Please enter the password once again for conformation</p>
+                <label>Password:</label>
+                <input type="password" name="cnpwd"><br><br><br><br>
+                    <button type="submit"> Signup</button>
+            </form>
+        </div>
+    </div>
+    </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment7/sign.html]: make proper indentation 
make sure it before pushing and check that all diva have closing tag for them

File: urmila/HTML/Assignment8/Assignment8.html [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+                    voluptatibus, quia, quos excepturi modi atque, repellat aliquid qui. Esse incidunt tenetur veritatis! Lorem, ipsum dolor sit amet
+                    consectetur adipisicing elit. Nulla distinctio saepe consequuntur ex repudiandae nihil iste dolorum voluptatibus, quia, quos excepturi modi
+                    atque, repellat aliquid qui. Esse incidunt tenetur veritatis! Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla distinctio
+                    saepe consequuntur ex repudiandae nihil iste dolorum voluptatibus, quia, quos excepturi modi atque, repellat aliquid qui. Esse incidunt
+                    tenetur veritatis!
+                </p>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment8/Assignment8.html:60]: you have to use here float make correct and push again

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
check all files before pushing on git

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**